modules/t/test-genome-DBs/homo_sapiens/variation/meta.txt

```diff
@@ -44,3 +44,7 @@
 44	\N	patch	patch_96_97_c.sql|add an unique index on the name column
 45	\N	patch	patch_97_98_a.sql|schema version
 46	\N	patch	patch_98_99_a.sql|schema_version
+47	\N	patch	patch_98_99_b.sql|Add the column data_source_attrib in the table variation_citation
+48	\N	patch	patch_98_99_c.sql|Increase the size of the title and doi columns in the publication table
+49	\N	patch	patch_98_99_d.sql|add key data_source_attrib
+50	\N	patch	patch_98_99_e.sql|Fix attrib ids in table variation_citation
```

Comment:
Variation have already released _patch_99_100_a.sql_ yet it doesn't appear here, nor anywhere else in this PR. Could you update _ensembl-variation_ (as well as _ensembl-compara_ to be sure, even though as far as I can see Compara haven't updated their schema yet) to their current master head and run the patcher again?